src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -117,10 +170,30 @@ public String getPageStreamingDescriptorName(Method method) {
     return varName(Name.upperCamel(method.getSimpleName(), "PageStreamingDescriptor"));
   }
 
+  public String getPageStreamingDescriptorConstName(Method method) {
+    return inittedConstantName(Name.upperCamel(method.getSimpleName()).join("page_str_desc"));
+  }
+
+  public String getBundlingDescriptorConstName(Method method) {
+    return inittedConstantName(Name.upperCamel(method.getSimpleName()).join("bundling_desc"));
+  }
+
   public void addPageStreamingDescriptorImports(ModelTypeTable typeTable) {
     // do nothing
   }
 
+  public void addBundlingDescriptorImports(ModelTypeTable typeTable) {
```

Comment:
I can see that, and I don't have a super strong opinion. The counter argument would be that while other methods can emit "Not Implemented" strings, these void-returning methods have no way to indicate that they aren't doing any work. So someone implementing a new language may not realize they need to implement the methods, because all that will go wrong is that imports will be missing.